src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
```

Comment:
Stylesheet with an `.scss` extension can't be linked directly in HTML. You should compile SCSS to CSS and link the compiled `.css` file instead.